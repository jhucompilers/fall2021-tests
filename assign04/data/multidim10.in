78 8 27 83 84 37 53 1 41 86 
10 75 47 66 94 42 17 62 16 5 
23 91 92 47 44 85 85 13 1 97 
15 55 96 51 46 59 23 76 98 67 
59 47 67 71 54 89 39 44 23 70 
37 51 7 51 7 54 55 75 6 64 
1 82 4 67 49 5 17 85 22 60 
40 52 92 47 48 19 36 71 55 13 
56 0 36 4 92 11 65 40 6 91 
75 66 50 95 56 21 76 35 50 59 

61 18 96 37 9 95 81 73 42 22 
27 16 77 72 40 12 2 81 65 23 
5 77 5 4 2 38 77 81 22 21 
42 27 86 45 61 99 53 24 95 90 
69 13 97 81 55 13 93 0 59 30 
92 24 56 75 89 79 57 45 27 44 
30 57 4 73 45 27 6 84 24 89 
26 43 73 56 32 15 17 8 29 8 
58 23 40 33 79 66 82 11 11 65 
54 73 97 90 31 23 98 56 74 87 
