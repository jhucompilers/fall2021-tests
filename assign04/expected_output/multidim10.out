26433
18117
35864
29902
22374
26894
34858
21913
26346
28957
19312
13379
29835
25383
20286
17270
22171
16466
22031
17024
25334
25343
33483
35521
24210
23500
30876
34142
27867
29994
25592
23657
34366
30544
26705
25811
35093
24575
24261
27031
27527
21482
37024
32449
24917
28318
33618
27051
26986
27120
18666
15955
27746
26154
18022
18277
18439
19923
20036
20763
16186
13714
30198
25443
17607
13462
18177
14582
22406
18555
18156
18304
26854
22609
18462
20664
25673
21492
19384
18992
19376
17554
27687
26046
14359
13993
27183
18537
18769
20004
25669
21347
37892
32581
24478
28843
32430
28614
29191
30799
